lib/cryptoexchange/exchanges/ninecoin/services/pairs.rb

```diff
@@ -5,6 +5,7 @@ class Pairs < Cryptoexchange::Services::Pairs
         PAIRS_URL = "#{Cryptoexchange::Exchanges::Ninecoin::Market::API_URL}/cy_mark_list"
 
         def fetch
+          return [] # API has moved or been taken down, docs are completely wrong
```

Comment:
Ok we can drop support. Perhaps documented in the README table would be good as well